### DISCUSSION TITLE
Actions Disabled after Incident on 21st February

### DISCUSSION THREAD

**Discussion by mihailomilosevic2001** [community/community#110270] [Actions]
### Select Topic Area

Bug

### Body

On 21st February, actions were disabled on my account. This error occurred around the time GitHub changed configuration and experienced a problem, so they reverted the changes. Can someone react on these issues promptly, as this error disrupts completely the workflow of the feature implementations.

**Comment by grandmas-favorite** [comment]
Hi @mihailom-db!

I'm sorry to hear that happened to you - has the issue occurred since?
